From b3c7debd83bacac7b076381f8ad4f731391004df Mon Sep 17 00:00:00 2001
From: Eduardo Chappa <chappa@washington.edu>
Date: Tue, 30 Mar 2021 09:35:20 -0600
Subject: [PATCH]    * Added line "#include <time.h>" to file
 imap/src/osdep/unix/os_osx.h      to fix a compilation error in mac, reported
 by several people. The fix      was tested successfuly by Steven Michaud in
 the following systems:        macOS 11.2.3             XCode 12.4       
 macOS 10.15.7            XCode 11.1        macOS 10.14.6            XCode
 10.3        macOS 10.13.6            XCode  9.1        macOS 10.12.6         
   XCode  8.3.3        OS X  10.11.6            XCode  7.3.1        OS X 
 10.10.5            XCode  6.1.1        OS X  10.9.5             XCode  6.1.1 
     See https://trac.macports.org/ticket/61590

---
 imap/src/osdep/unix/os_osx.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git imap/src/osdep/unix/os_osx.h imap/src/osdep/unix/os_osx.h
index 6fcb014da959..558106e9907e 100644
--- imap/src/osdep/unix/os_osx.h
+++ imap/src/osdep/unix/os_osx.h
@@ -32,7 +32,7 @@
 #include <utime.h>
 #include <syslog.h>
 #include <sys/file.h>
-
+#include <time.h>
 
 /* Mac OS X gets this wrong as of Leopard */
 
-- 
2.31.1

